plugins/ata/README.md

```diff
@@ -0,0 +1,38 @@
+ATA
+===
+
+Introduction
+------------
+
+This plugin adds support for ATA/ATAPI storage hardware. Devices are enumerated
+from the block devices and if ID_ATA_DOWNLOAD_MICROCODE is supported they can
+be updated with appropriate firmware file.
+
+Updating ATA devices is more dangerous than other hardware such as DFU or NVMe
+and should be tested carefully with the help of the drive vendor.
+
+The device GUID is read from the trimmed model string.
+
+Firmware Format
+---------------
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+an unspecified binary file format.
+
+GUID Generation
+---------------
+
+These device use the ATA DeviceInstanceId values, e.g.
+
+ * `ATA\VEN_1179&DEV_010F&REV_01`
+ * `ATA\VEN_1179&DEV_010F`
+ * `ATA\VEN_1179`
+
```

Comment:
Missing the update protocol in the readme.

Reply:
Ohh? What should we say about the update protocol? I don't think we've documented it in other plugins, right?

Reply:
It's in our new [plugin checklist](https://github.com/hughsie/fwupd/wiki/New-plugin-checklist).  I think just mention what it is so that people don't have to dig into the code.

Reply:
Yeah